.github/workflows/ci_weekly_report.yaml

```diff
@@ -0,0 +1,125 @@
+name: weekly CI test report
+on:
+  schedule:
+    - cron: '37 9 * * 1' # 9:37AM UTC -> 2:37AM PST every monday
+  workflow_dispatch:
+    inputs:
+      ci_runs:
+        default: '100'
+        description: 'The amount of runs to trigger in CI test report'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  setup:
+    if: github.repository == 'commaai/openpilot'
+    runs-on: ubuntu-latest
+    outputs:
+      ci_runs: ${{ steps.ci_runs_setup.outputs.value }}
+    steps:
+      - id: ci_runs_setup
+        run: |
+          mylist="value=["
+
+          for i in $(seq 1 ${{ inputs.ci_runs }});
+          do
+              if [ $i != ${{ inputs.ci_runs }} ]; then
+                  mylist+="\"$i\", "
+              else
+                  mylist+="\"$i\""
+              fi
+          done
+          mylist+="]"
+
+          echo "$mylist" >> $GITHUB_OUTPUT
```

Comment:
there isn't a simpler way to write this?

Reply:
From my time looking into how to make dynamic matrix values, this was really the only way to do it. Felt a bit hacky and since the requirement is having the ability to run `workflow_dispatch` with any amount of runs inputted this logic would allow us to do just that. If we decided to just stick with a static value of runs we can cut this down.